### ISSUE TITLE
specify that collection should be included in synthesis

### ISSUE THREAD

**Issue by kcranston** [OpenTreeOfLife/opentree#947]
We now have ~ 20 collections and it is not clear which ones should be included in synthesis. From the Florida workshop, only 1 of the three is ready for inclusion in synthesis (and I only know that because I emailed the collaborators).

We probably want a way for users to specify:

* 'yes, use this collection for synthesis' or 'no, this collection is not for synthesis'
* maybe also a way to say 'use *this version* of this collection for synthesis'.

**Comment by nfranz** [comment]
I would agree - a version- and group-identifying "yes, no" option, where a later version would supersede an earlier one (although group of users may internally vary by then).

**Comment by jar398** [comment]
A different way to specify this would be by allowing collections of collections, as requested here: https://github.com/OpenTreeOfLife/opentree/issues/930

**Comment by jimallman** [comment]
> A different way to specify this would be by allowing collections of collections

This makes sense to me, since it provides an easy way to "nominate" a collection for synthesis (by simply adding/updating it in the main collection).

**Comment by jimallman** [comment]
Just a reminder that in addition to the "include this?" decision, there's the question of ranking. The collection-of-collections handles this naturally.

**Comment by kcranston** [comment]
As we get more and more collections (~12 new ones this weekend!) it is getting harder to organize which ones should be in synthesis. Can we start to estimate what's required for keeping track of synthesis status?

Requirements:

* user can see synthesis status (included in current synthesis? proposed for synthesis?)
* user can propose a collection for synthesis
